app/web/markdown/blog/2024/09/08/summer-2024-updates.md

```diff
@@ -0,0 +1,98 @@
+
+We have always believed that in building this community, it's important that we set clear expectations for the community, such as via our Community Guidelines. By taking an opinionated stance on these issues and keeping up a system of careful moderation, we believe we can create the safest and healthies couch surfing community.
+
+
+## Changes to our Board of Directors
+
+Couchers.org is operated by Couchers, Inc., a 501(c)(3) non-profit organization run entirely by volunteers. As part of future-proofing the project, we have invited some new people onto our Board of Directors. The current Board Members are the Founders Aapeli ([@aapeli](https://couchers.org/user/aapeli)) and Itsi ([@itsi](https://couchers.org/user/itsi)), as well as at-large Board Members Emily ([@emily](https://couchers.org/user/emily)), Yannic ([@spreeni](https://couchers.org/user/spreeni)), Natalia ([@natalia](https://couchers.org/user/natalia)), Paul ([@paul](https://couchers.org/user/paul)), and Jesse ([@jesse](https://couchers.org/user/jesse)). These folks have all been involved with the project for a long time and the outgoing board is firm in their belief as good custodians of the project with the right spirit and vision for our community.
+
+You can read more about these wonderful community members and other minor governance changes on the [blog post announcing the changes](https://couchers.org/blog/2024/04/30/new-board-members).
+
+
+## Past volunteers: get a badge on your profile!
+
+Couchers has been built by an incredible army of volunteers, to whom our whole community is incredibly grateful for. We want to recognize and celebrate this the invaluable contributions every past developer has made to the project.
+
+If you have volunteered and would like to be recognized for it on your profile, please fill in the following form and we'll add one to your profile:
+
+[Past Volunteer form](https://forms.gle/UYdBHn8S1tCNiVgNA)
+
+
+## Support Couchers: volunteer or donate!
+
+The Couchers.org platform and community is built by a team of volunteers in their spare time, organized around a non-profit. All our work is open source and community lead. You too can support the project.
+
+We are currently raising money for development tools to build mobile apps for Android and iOS. This is an expensive endeavour, so we are looking for some extra financial support from the community. We are also hoping to produce some merchandise, which has some upfront costs.
+
+[Donate to Couchers](https://couchers.org/donate)
+
+
+### Volunteer
+
+We are looking for self-driven and motivated volunteers to help build community or develop the platform. We do not currently have a great recruiting system, so we are looking for people who are able to join the project and take charge of individual projects or initiatives.
```

Comment:
```suggestion
We are looking for motivated and self-driven volunteers to help build out the community or develop the platform. Our recruiting system is currently quite limited, so we are looking for people who are able to join the project and take charge of individual projects or initiatives.
```

Reply:
great. adopted but changed to "Our recruiting resources are currently quite limited"